src/hotspot/cpu/riscv/nativeInst_riscv.hpp

```diff
@@ -383,12 +416,6 @@ class NativeMovConstReg: public NativeInstruction {
   intptr_t data() const;
   void set_data(intptr_t x);
 
-  void flush() {
```

Comment:
why the removal?

Reply:
It was unused, and I had review comment in there.
It was suggested to be removed instead of fixing the comment.